ui/packages/app/src/project_setting/MetadataSetting.js

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from "react";
+import MetadataList from "./metadata/MetadataList";
+import { ProjectsContext } from "@caraml-dev/ui-lib";
+import { EuiLoadingChart, EuiTextAlign } from "@elastic/eui";
+
+const MetadataSetting = () => {
+  const { currentProject, refresh } = useContext(ProjectsContext);
+
+  return (
+    <>
+      {!currentProject ? (
+        <EuiTextAlign textAlign="center">
+          <EuiLoadingChart size="xl" mono />
+        </EuiTextAlign>
+      ) : (
+        <MetadataList project={currentProject} fetchUpdates={refresh} />
```

Comment:
`List` is misleading, its not a table nor a table

Reply:
Would `ProjectInfoSelection` be okay?